draft-ietf-rats-architecture.md

```diff
@@ -743,19 +750,9 @@ in order to bootstrap the sequence.
 
 ## Verifier
 
-The Verifier trusts (or more specifically, the Verifier's security
-policy is written in a way that configures the Verifier to trust) a
-manufacturer, or the manufacturer's hardware, so as to be able to
-appraise the trustworthiness of that manufacturer's devices.  In solutions
-with weaker security, a Verifier might be configured to implicitly
-trust firmware or even software (e.g., a hypervisor).  That is, it
-might appraise the trustworthiness of an application component, operating
-system component, or service under the assumption that information
-provided about it by the lower-layer hypervisor or firmware is true.
-A stronger level of assurance of security comes when information can be vouched
-for by hardware or by ROM code, especially if such hardware is
-physically resistant to hardware tampering.  The component that is
-implicitly trusted is often referred to as a Root of Trust.
+The Verifier trusts (or more specifically, the Verifier's security policy is written in a way that configures the Verifier to trust) an Endorser who supplies an Endorsement which contains information by which the Verifier can come to trust the Attester.
+Typically, that information is a key used to verify the signature on Attestation Evidence which proves that the Attester has the private key that is only put into trustworthy devices by the Endorser.
+That is, the Verifier comes to trust the Attester transitively through the Endorser.
```

Comment:
Hard to review since line breaks were removed.  Please revert line breaks.

Reply:
Other than line breaks, I agree with the new content.